libbs/decompilers/ghidra/interface.py

```diff
@@ -34,8 +34,9 @@ def _ghidra_transaction(self: "GhidraDecompilerInterface", *args, **kwargs):
 
 
 class GhidraDecompilerInterface(DecompilerInterface):
-    def __init__(self, loop_on_plugin=True, **kwargs):
+    def __init__(self, loop_on_plugin=True, start_headless_watchers=False, **kwargs):
         self.loop_on_plugin = loop_on_plugin
+        self.start_headless_watchers = start_headless_watchers
```

Comment:
Remember if something is a private variable, i.e it should never be accessed and changed by some outside process using this object, than it should start with an `_`. Maybe rename this variable to be private?

Reply:
@mahaloz wouldn't the user set this when creating the DecompilerInterface object though?

Reply:
they would set the parameter that is incoming to the Deci. They would not set the actual attribute like `deci.start_headless_watchers = True`, because if we allowed that it would make it seem that we have some kind of always-on checker to see when you assigned a bool there.